haskell/nettest/Nettest.hs

```diff
@@ -126,24 +127,27 @@ scNettestScenario originateTransferlist transferlistType = uncapsNettest $ do
 
     configureMinter' :: TAddress Parameter -> Address -> Address -> Maybe Natural -> Natural -> capsM ()
     configureMinter' sc' from for expectedAllowance newAllowance =
-      callFrom
-        (AddressResolved from)
+      withSender
+        (AddressResolved from) $
+        call
         sc'
         (Call @"Configure_minter")
         (ConfigureMinterParam for expectedAllowance newAllowance)
 
     removeMinter :: Address -> Address -> capsM ()
     removeMinter from whom =
-      callFrom
-        (AddressResolved from)
+      withSender
+        (AddressResolved from) $
```

Comment:
So... there's a bunch more cases of the two suggestions above in this file :sweat_smile: 

The suggestion is to move the first argument of `withSender` to its line and add indentation to the arguments of `call`.
Note that in many cases where now `call`  can fit with its arguments on a single line of ~80 and I think we should do that where possible.

Reply:
Looks like I've updated everything now